Tokenize/test/lexer.jl

```diff
@@ -296,205 +296,220 @@ end
 @testset "show" begin
     io = IOBuffer()
     show(io, collect(tokenize("\"abc\nd\"ef"))[2])
-    @test String(take!(io)) == "1,2-2,1          STRING         \"abc\\nd\""
+    @test String(take!(io)) == "1-5        STRING         "
 end
 
-~(tok::T.AbstractToken, t::Tuple) = tok.kind == t[1] && untokenize(tok) == t[2]
+~(tok::T.Token, t::Tuple) = tok.kind == t[1] && untokenize(tok, t[3]) == t[2]
```

Comment:
Oho. I see what you did there.  Very tricky.

Maybe neater to just define a utility function to take the place of `collect(tokenize(ts))` instead which yields the tuples I'm testing against below. Then use `==` rather than having to pass `str` everywhere.

(Nevertheless, let's merge this as-is; that could be cleaned up separately.)